app.js

```diff
@@ -14,12 +14,41 @@ app.use(express.static(path.join(__dirname, 'public')));
 
 // Register the location for handlebars partials here:
 
-// ...
+hbs.registerPartials(path.join(__dirname, 'views/partials'));
 
 // Add the route handlers here:
 
 app.get('/', (req, res) => {
   res.render('index');
 });
 
+app.get('/beers', (req, res) => {
+  punkAPI
+    .getBeers()
+    .then(beersFromApi => {
+      /*     console.log('Beers from the database: ', beersFromApi) */
+      res.render('beers', { beers:beersFromApi });
+    })
+    .catch(error => console.log(error));
+});
+
+//TEST to get ID
+app.get('/beer/:id', (req, res) => {
+  const beerId = req.params.id;
+  punkAPI.getBeers(beerId).then(responseFromAPI => {
+    res.render('beer/id', {random:responseFromAPI });
```

Comment:
You're using `punkAPI.getBeers` which is the same method that you used in the `/beers` route and this method returns you all the beers. But you just want one specific beer, so in the lab instruction they tell you that you have another method for this that is `punkAPI.getBeer(id)`:

```
  punkAPI.getBeer(beerId).then(responseFromAPI => {
```

Since here you want to render just one specific beer you can reuse the `random-beer.hbs` page and pass the response from the API with the same name:

```
    res.render('random-beer', { responseFromAPI });
```